shared/common-adapters/standard-screen.desktop.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './standard-screen'
+
+const defaults = {
+  showClose: true,
+}
+
+const StandardScreen = (props: Props) => {
+  const {children, onClose, showClose, errorMessage} = {...defaults, ...props}
+  return (
+    <Box style={styleContainer}>
+      {!!onClose && showClose && <Icon style={styleClose} type='iconfont-close' onClick={() => onClose()} />}
+      {errorMessage && <Box style={styleErrorBanner}><Text style={styleErrorBannerText} type='BodySmallSemibold'>{errorMessage}</Text></Box>}
+      <Box style={styleContentContainer}>
+        {children}
+      </Box>
+    </Box>
+  )
+}
+
+const styleContainer = {
+  ...globalStyles.flexBoxColumn,
+  flex: 1,
+  alignItems: 'center',
+  position: 'relative',
+  paddingTop: globalMargins.large,
+  paddingBottom: globalMargins.large,
+  ...globalStyles.scrollable,
```

Comment:
It seems like most of the code places these object spread inclusions at the top. Any particular reason you didn't do so in this file?